### ISSUE TITLE
real StainGAN citation -Technische Universität München

### ISSUE THREAD

**Issue by xtarx** [alexmomeni/StainGAN#1]
Hi alexmomeni,

I am Tarek the co-author of StainGAN, thank you for re-implmenting the method and I hope you found it benficial.

One thing, Can you please cite the paper and the real repository of the code in the Github readme and on every code you share else where to avoid any legal problems.

Thanks for understanding

https://github.com/xtarx/StainGAN

```
@article{shaban2018staingan,
  title={StainGAN: Stain Style Transfer for Digital Histological Images},
  author={Shaban, M Tarek and Baur, Christoph and Navab, Nassir and Albarqouni, Shadi},
  journal={arXiv preprint arXiv:1804.01601},
  year={2018}
}
```

**Comment by luqioayun** [comment]
Hello Tarek,
My name is Lu Qiaoyun and I am a student from Beijing University of Posts and Telecommunications in China.
I have read the paper "StainGAN: Stain Style Transfer for Digital Histological Images" and I have two questions about it.

First: In the section "Implementation of 3.2 Use-Case Application" , as the paper describes "For the stain normalization, representative reference slide form lab 2 was picked for the conventional methods, Our method was trained using the same parameters from the previous experiment on training set: 68000 patches from both labs. " Are the 68000 patches generated from "Camelyon16 challenge(https://camelyon16.grand-challenge.org/)",the data for the classification evaluation?

Second, I download the data from "https://camelyon16.grand-challenge.org/", but there is no labs information. Do you know it?

Thanks a lot!

My information:
Name: Lu Qiaoyun
e-mail: 1182102020@qq.com
